.github/workflows/notify_codeowners.yml

```diff
@@ -18,6 +18,7 @@ jobs:
       - name: Drop a message for codeowners
         env:
           PR: ${{ steps.findPr.outputs.pr }}
+          GITHUB_BOT_TOKEN: ${{ secrets.GITHUB_BOT_TOKEN }}
```

Comment:
Need to modify the secret name to not prefix `GITHUB_`:
https://docs.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets#naming-your-secrets